### PR TITLE
Features/dashboard last invoice add link to read update invoice url 376

### PR DESCRIPTION
**Pull request by onfranciis**
https://github.com/idurar/idurar-erp-crm/assets/96952305/75aa4d5f-60ee-476a-bd37-43088a43c254

I refactored the quotes folder and added route feature for recent dashboard tables to address issue #376

### CODE REVIEW

**Review by salahlalami — APPROVED** [review]

File: frontend/src/modules/DashboardModule/components/RecentTable/index.jsx
Comment (Ando22) [inline frontend/src/modules/DashboardModule/components/RecentTable/index.jsx:1]: I see on your changes there are `QuoteModules` and `.QuoteModules`. can you please combine those modules ?

File: frontend/src/pages/Quote/index.jsx [hunk 1]
```diff
@@ -1,14 +1,13 @@
-import React from 'react';
 import dayjs from 'dayjs';
 import { Tag } from 'antd';
 
 import QuoteModule from '@/modules/QuoteModule';
```
Comment (Ando22) [inline frontend/src/pages/Quote/index.jsx:4]: This code now is broken. can you please fix this ?

Screen Shot 2023-09-04 at 18 46 18

**Review by Ando22 — CHANGES_REQUESTED** [review]

File: frontend/src/modules/DashboardModule/components/RecentTable/index.jsx
Comment (onfranciis) [reply frontend/src/modules/DashboardModule/components/RecentTable/index.jsx:1]: My bad, it's not supposed to exist. I have deleted the folder.

File: frontend/src/pages/Quote/index.jsx (hunk 1 above)
Comment (onfranciis) [reply frontend/src/pages/Quote/index.jsx]: Fixed, kindly restart your server and try again

**Review by Ando22 — APPROVED** [review]

**Pull request merged**